models/staging/edfi_3/stage/stg_ef3__student_assessment_education_organizations.sql

```diff
@@ -0,0 +1,39 @@
+with base as (
+    select * from {{ ref('base_ef3__student_assessment_education_organizations') }}
+    where not is_deleted
+),
+
+
+keyed as (
+    select
+        {{
+            dbt_utils.generate_surrogate_key(
+                [   'tenant_code',
+                    'api_year',
+                    'ed_org_identifier',
+                    'ed_org_association_type',
+                    'assessment_identifier',
+                    'student_assessment_identifier',
+                    'student_unique_id',
+                    'lower(namespace)'
+                ]
+            )
+        }} as k_student_assessment_ed_org,
+        {{ gen_skey('k_ed_org') }},
```

Comment:
use ed_org macro instead so we can pull out both k_school k_lea https://github.com/edanalytics/edu_edfi_source/blob/5884ee1975085fa81d3f249342569ae95fe47c64/models/staging/edfi_3/stage/stg_ef3__courses.sql#L13